### ISSUE TITLE
Load_predictor() error

### ISSUE THREAD

**Issue by lamechah** [amaiya/ktrain#264]
Hi, I'm trying to load the model I saved into a new colab file and I'm getting this error: 

**Call to keras.models.load_model failed.  Try using the learner.model.save_weights and learner.model.load_weights instead.**
Error was: SavedModel file does not exist at: /content/drive/My Drive/research/BERT_multiclass/{saved_model.pbtxt|saved_model.pb}
---------------------------------------------------------------------------

This is my code: https://colab.research.google.com/drive/1tU1-ms07DsAYbRIdTDLyFufCsSHAZ-ox?usp=sharing
This is how the folder with the model looks like:  
Folder: BERT_multiclass
Files: tf_model.preproc, tf_model.h5

**Comment by amaiya** [comment]
I believe this may be some sort of intermittent issue or bug with Google Colab, rather than **ktrain**.

When I received this error on Colab, I simply tried loading the predictor again and it worked.  After that, I wasn't able to reproduce the error even when starting an entirely new Colab session and retracing the same steps including remounting the drive.

Can you please try re-running the `load_predictor` command after you receive the initial error in the same Colab session and see if it works?

**Comment by lamechah** [comment]
Hi @amaiya , thank you very much for the help.

 I tried re-running the load_predictor in the same session but I'm still getting the error. I also tried running the same code in a jupyter notebook and I'm having the same issue: (attaching ss) 

Could this have to do with the fact that I'm running the load_predictor in a completely new python file? 

![image](https://user-images.githubusercontent.com/38266737/95698942-84e9d500-0c86-11eb-9014-ee9f8d0d2593.png)
![image](https://user-images.githubusercontent.com/38266737/95698965-9c28c280-0c86-11eb-81ce-514dd105dbfc.png)

**Comment by amaiya** [comment]
What versions of TensorFlow, Python, and `transformers` are you using?

**Comment by lamechah** [comment]
Yes, exactly! I'm mean I'm also getting this error outside of Colab and Drive.

Versions: 
Tensorflow:  2.3.1 
Python: 3.7.3
Transformers: 3.1.0

**Comment by amaiya** [comment]
I'm not able to reproduce this problem outside of Google Colab/Drive.  What are the contents of `predictor_test` folder?

What happens when you do this?

```python
from transformers import *
model = TFAutoModelForSequenceClassification.from_pretrained('predictor_test')
```

**Comment by lamechah** [comment]
Good news! This is working for me now without that import statement :) 
I'm not exactly sure what the issue was, but I think it might be related to the package versions. 
How I fixed it? 
1.I pip uninstalled and re-installed everything 
2. Ignored the error message regarding incompatible versions:
ERROR: seqeval 1.0.0 has requirement numpy==1.19.2, but you'll have numpy 1.18.5 which is incompatible.
ERROR: seqeval 1.0.0 has requirement scikit-learn==0.23.2, but you'll have scikit-learn 0.22.2.post1 which is incompatible.
3. Included same import statements that I used for training model
